io.catenax.market_place_offer/2.0.0/MarketplaceOffer.ttl

```diff
@@ -0,0 +1,413 @@
+   samm:exampleValue true .
+
+:mechanicalDamage a samm:Property ;
+   samm:preferredName "mechanical damage"@en ;
+   samm:description "Information on mechanical damage to the part, e.g. two broken ribs or part deformation indicate mechanical damage."@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .
+
+:corroded a samm:Property ;
+   samm:preferredName "corroded"@en ;
+   samm:description "Information on whether the product has corrosion."@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .
+
+:discolored a samm:Property ;
+   samm:preferredName "discolored"@en ;
+   samm:description "Information on whether the product has been discoloured."@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .
+
+:availabilityDate a samm:Property ;
+   samm:preferredName "availability date"@en ;
+   samm:description "The availability date when the product is in stock."@en ;
+   samm:characteristic :AvailabilityDateCharacteristic ;
+   samm:exampleValue "2022-03-11"^^xsd:date .
+
+:mileage a samm:Property ;
+   samm:preferredName "mileage"@en ;
+   samm:description "The mileage the item was ins use."@en ;
+   samm:characteristic :MileageCharacteristic ;
+   samm:exampleValue "120000.06"^^xsd:decimal .
+
+:burned a samm:Property ;
+   samm:preferredName "burned"@en ;
+   samm:description "Information on whether the product has been fired and suffered damage from heat."@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .
+
+:errorCodes a samm:Property ;
+   samm:preferredName "Error Codes"@en ;
+   samm:description "Error codes that are recorded as part of onboard data and can be read as part of diagnostics"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Engine control    Fault" .
+
+:partMarking a samm:Property ;
+   samm:preferredName "Part Marking"@en ;
+   samm:description "Unique identifier for a core"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "ABCD" .
+
+:vehicleDescription a samm:Property ;
+   samm:preferredName "Vehicle Description"@en ;
+   samm:description "Description of vehicle where the EoL part has been dismantled from (if applicable)"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "VW (VOLKSWAGEN) Golf VII 1.4 TSI 1.4 103.0kw" .
+
+:serialNumber a samm:Property ;
+   samm:preferredName "serial number"@en ;
+   samm:description "Unique identifier of an item in a serial production"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "NO-053448509504856168886009" .
+
+:taxPercentage a samm:Property ;
+   samm:preferredName "tax percentage"@en ;
+   samm:description "Tax percentage applicable to the product"@en ;
+   samm:characteristic :Decimal ;
+   samm:exampleValue "10.5"^^xsd:decimal .
+
+:dimensions a samm:Property ;
+   samm:preferredName "dimensions"@en ;
+   samm:description "(length, height, width, diameter, weight)"@en ;
+   samm:characteristic ext-dimension:PhysicalDimensionsCharacteristic .
```

Comment:
please check if you are able to use `https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.shared.quantity/1.0.0/Quantity.ttl` model here.

Reply:
If it is linked to Quantity aspect, the property "dimensions" has both mass and linear characteristic. However I cannot link a single property to 2 characteristics. Do you have a solution here? Or should I still use the physical dimension aspect?

Reply:
Quantity: You can link it directly to each specific characterisitc (for reference i did the same in the digital product passport). For weight to the massCharacteristic, for the others each to the linear characteristic.
(Physical Dimensions: This model is very restrictive (as of now) and the unit can not be adjusted.)

Reply:
Thanks @catroest for helping here.
@yuchengluo1 here's the example https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.generic.digital_product_passport/3.0.0/DigitalProductPassport.ttl#L665.